docs/notes/v7.0.0.md

```diff
@@ -0,0 +1,25 @@
+# MinIO Operator v7.0.0 release notes
+
+## About this release
+
+MinIO Operator is the recommended production-ready way to manage MinIO deployments in Kubernetes. This version includes a new base version requirements, deprecations, bugfixes and more.
+
+## What’s new?
+
+Operator 7.0.0 is now available. This release requires Kubernetes version 1.29.0 or later.
+If your cluster does not use Kubernetes version 1.29.0 or later, you must upgrade your cluster to before installing or upgrading to Operator v7.0.0+.
+
+Some breaking changes motivates a mayor release, here is the list of breaking changes:
+
+* remove minioJob
```

Comment:
What is the migration/upgrade path here?

What needs to change on an existing tenant for the update to work?

Or are these not expected to break workflows if  you have a tenant using them?

Reply:
For the Tenant CRD is as simple as update the Operator version, it is not going to be disruptive at all.
For the MinIO Job there is no replacement in the Operator, AIStor have a similar functionality for enterprise, the AdminJob, I would say the recommendation if MinIOJob is needed, to move to the AIStor operator

Reply:
So if someone has an existing MinioJob YAML its just going to stop working with 7.0.

FOr PDB same - do they have to scrub PDB from tenant first?

Reply:
right, MinIOJob will stop working on 7.0.0

Reply:
MinIOJob is a one-time operation.

@pjuarezd Does Helm remove the CRD during upgrade? If not then we may want to add code to deal with this.

Reply:
yes, the Helm upgrade will remove the CRD @ramondeklein

Reply:
So no other changes are required for upgrade? An existing tenant w/ PDB settings should still work in 7.0.0 - or are we asking them to scrub PDB settings _first_ @pjuarezd